// app/prepareAppContainer.js
var $ = require('jquery');

module.exports=  function(){
    $(function(){
        var $container= $('<div>').attr('class','app');

        var $table=$('<table>');
        var size=6;

        var board = [];

        for(var x=0;x<size;x++){
            board[x]=[];//wiersz
            for (var y=0; y<size;y++){
                board[x][y]={};//kolumna
            }
        }

        board[5][3]={player:1};

        function stylePlayers($cell, cell){
            $cell.css('backgroundColor',cell.player===1 ? '#0ff':'')
        }

        function createTable(board, style){
            var $table=$('<table>');
            var size=board.length;


            for (var y =0;y<size;y++){
                var $row=$('<tr>');
                for (var x=0;x<size;x++){
                    var $cell = $('<td>');
                    $cell.css({
                        border: '1px solid black',
                        width: 50,
                        height: 50,
                        backgroundColor: '#fff',
                    });
                    style($cell,board[x][y]);
                    $row.append($cell);
                }
                $table.append($row);
            }
            return $table;
        }

        var n=0;
        $('body').on('load click', function(){



        })
        $container.append(createTable(board, stylePlayers));

        $('body').append($container);

    });
};
